tests/test_store_manager_sales.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from app import create_app
 import json
-from baseTest import BaseTestClient
+from .baseTest import BaseTestClient
 
 class TestSale(BaseTestClient):
     def test_get_admin_sales(self):
```

Comment:
What are you testing in this test?

Reply:
Testing whether the admin can get all sales in the store.